can_spec_my18.yml

```diff
@@ -686,6 +676,19 @@ buses:
             position: 60
             length: 1
             c_type: bool
+      DashRequestLC:
+        can_id: 0x0D5
+        is_big_endian: true
+        period: 69ms
+        segments:
+          using_launch_ctrl:
+            position: 0
+            length: 1
+            c_type: bool
+          launch_ctrl_slip_ratio: # Ranges from 105 to 115
```

Comment:
Why does it range between these values? Will this ever change?

Reply:
I talk about this is confluence docs. This number was something that Luis suggested but could use some more backing.

Reply:
Yeah, might be good to at least show how you might come up with this on confluence.